### PR TITLE
[RAC5548]Add timeout and retry to deploy ova

### PR DESCRIPTION
**Pull request by PengTian0**
MasterCI #260 failed because it stuck at 99% during deploy ova.
The PR adds timeout and retry to the step.
Test job:
http://10.**.**.175:8080/job/MasterCI_Tian/
@panpan0000 @changev @iceiilin @anhou

### CODE REVIEW

File: shareMethod.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  set +e
+  # $1 command to execute
+  # $2 timeout
+  # $3 retries on timeout
+  if [ -z "${1}" ]; then
+     echo "execWithTimeout() Command not specified"
+     exit 2
+  fi
+  cmd="/bin/sh -c \"$1\""
+  #timeout default to one minute
```
Comment (panpan0000) [inline shareMethod.sh]: wrong comment

File: shareMethod.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (panpan0000) [inline shareMethod.sh:1]: please try to use local for variable declaration in a shared method

Comment (PengTian0) [reply shareMethod.sh:1]: Thanks!
Fixed.

File: shareMethod.sh (hunk 1 above)
Comment (PengTian0) [reply shareMethod.sh]: Thanks!
Fixed.

**Review by panpan0000 — APPROVED** [review]

**Pull request merged**